misk-hibernate/src/test/kotlin/misk/hibernate/VitessSchemaMigratorTest.kt

```diff
@@ -0,0 +1,80 @@
+package misk.hibernate
+
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.vitess.StartVitessService
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.sql.Connection
+import javax.inject.Inject
+
+@MiskTest(startService = true)
+internal class VitessSchemaMigratorTest {
+  @MiskTestModule
+  val module = MoviesTestModule()
+
+  @Inject @Movies lateinit var transacter: Transacter
+  @Inject lateinit var queryFactory: Query.Factory
+  @Inject @Movies lateinit var schemaMigrator: SchemaMigrator
+  @Inject @Movies lateinit var vitessService: StartVitessService
+
+  private val shard = Shard(Keyspace("movies"), "-80")
+
+  @BeforeEach fun createSchemaMigrationTable() {
+    schemaMigrator.initialize()
+  }
+
+  @AfterEach fun cleanUpMigrationTable() {
+    openDirectConnection().use { c ->
+      val schemaVersion = c.prepareStatement("""
+            |DELETE FROM `vt_movies_-80`.schema_version
+            |""".trimMargin())
+      schemaVersion.executeUpdate()
+    }
+  }
+
+  @Test fun availableMigrations() {
+    val migrations = schemaMigrator.availableMigrations(Keyspace("movies"))
+    assertThat(migrations.map { it.version }).containsAll(listOf(1, 2))
+  }
+
+  @Test fun appliedMigrations() {
+    assertThat(schemaMigrator.appliedMigrations(shard).map { it.version }).isEmpty()
+
+    insertSchemaMigration("1")
+    assertThat(schemaMigrator.appliedMigrations(shard).map { it.version }).contains(1)
+  }
+
+  @Test fun requireAll() {
+    schemaMigrator.initialize()
+
+    assertThat(schemaMigrator.appliedMigrations(shard).map { it.version }).isEmpty()
+
+    insertSchemaMigration("1")
+    insertSchemaMigration("2")
+    schemaMigrator.requireAll(shard)
```

Comment:
anything to assert here? Or is it enough that it doesn't throw an exception? Could we leave a comment indicating that?